gdcdictionary/schemas/analyte.yaml

```diff
@@ -111,11 +114,13 @@ properties:
     $ref:
       - "_terms.yaml#/analyte_volume/common"
     type: number
+    minimum: 0
```

Comment:
didn't spot this in the ticket (DICT-100), but it makes sense.

Reply:
confirmed with Bill as well